mlflow/entities/logged_model_status.py

```diff
@@ -12,6 +12,8 @@ class LoggedModelStatus(str, Enum):
     READY = "READY"
     FAILED = "FAILED"
 
+    FINALIZED_STATUSES = ["READY", "FAILED"]
```

Comment:
Can we define a static method instead? This looks like another Enum member.

Reply:
I defined this as a classmethod so that we can access it as a property. I also prefixed with underscore so that users don't see it, e.g. in the docs or list of suggestions for completions